### PR TITLE
Mute and unmute command

### PR DESCRIPTION
**Pull request by Nepgfurmixpro**
Adding a mute and unmute command. Everything should work. I have tested on my bot and it works just fine. #44

### CODE REVIEW

File: bot/commands/mute.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+const config = require('../../config.json');
+const Discord = require('discord.js');
+
+module.exports.execute = async (client, message, args) => {
+    if (message.member.roles.cache.has(config.bot.roles.staff)) {
+        if (args.length == 0) { await message.channel.send('Mention someone to mute!') } else {
+            let role = message.guild.roles.cache.get(config.bot.roles.muted)
+            let user = message.mentions.members.first()
```
Comment (Zaph-x) [inline bot/commands/mute.js:8]: The command fails if a user has not been mentioned. Maybe add a try catch?

File: bot/commands/unmute.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+const config = require('../../config.json');
+const Discord = require('discord.js');
+
+module.exports.execute = async (client, message, args) => {
+    if (message.member.roles.cache.has(config.bot.roles.staff)) {
+        if (args.length == 0) { await message.channel.send('Mention someone to unmute!') } else {
+            let role = message.guild.roles.cache.get(config.bot.roles.muted)
+            let user = message.mentions.members.first()
```
Comment (Zaph-x) [inline bot/commands/unmute.js:8]: See above comment

**Review by Zaph-x — CHANGES_REQUESTED** [review]
I have made some comments. I will merge the PR once these have been addressed

**Review by Zaph-x — APPROVED** [review]

File: bot/commands/unmute.js (hunk 2 above)
Comment (Nepgfurmixpro) [reply bot/commands/unmute.js:8]: Alright everything should be fixed

File: bot/commands/mute.js (hunk 1 above)
Comment (Nepgfurmixpro) [reply bot/commands/mute.js:8]: Alright everything should be fixed

**Pull request merged**